streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -197,6 +196,8 @@ public class TaskManagerTest {
     @Mock(type = MockType.STRICT)
     private Consumer<byte[], byte[]> consumer;
     @org.mockito.Mock
+    private Consumer<byte[], byte[]> mockitoConsumer;
```

Comment:
I ran into quite a lot of problems when I tried migrating the whole mock, so I decided to do the migration test-by-test. This way problems could be flushed out. By introducing this mock and using `setMainConsumer` on a test-by-test basis things are easier to go through (at least in my opinion 😊)